design/user_types.go

```diff
@@ -22,3 +23,34 @@ var UpdateWorkItemPayload = Type("UpdateWorkItemPayload", func() {
 	Attribute("version", Integer, "Version for optimistic concurrency control")
 	Required("type", "name", "fields", "version")
 })
+
+// CreateTrackerAlternatePayload defines the structure of tracker payload for create
+var CreateTrackerAlternatePayload = Type("CreateTrackerAlternatePayload", func() {
+	Attribute("url", String, "URL of the tracker")
+	Attribute("credentials", String, "Credential to access the tracker")
```

Comment:
Drop credentials. It opens up a large set of issues we don't really want to deal with atm. Public access trackers only for now.